sbi/utils/tensorboard_output.py

```diff
@@ -0,0 +1,213 @@
+"""Utils for processing tensorboard event data."""
```

Comment:
Could you copy the [license statement](https://github.com/mackelab/sbi/blob/0c0860f7c1e53343f71e314102bf9e2d99a85ac1/sbi/inference/base.py#L1-L2) to the beginning of the file?